frontend/pages/endpoint.tsx

```diff
@@ -43,7 +43,7 @@ const JsonExampleBlock = styled.code<JsonExampleBlockProps>`
   white-space: pre;
 `
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, react/prop-types
 function JsonExample({ data }: { [k: string]: any }): JSX.Element {
```

Comment:
Original error before the change shown below. This seemed like the easiest approach to resolve given the current function signature and type information, but certainly open to a non-disable alternative if anyone has one

```
/home/circleci/project/frontend/pages/endpoint.tsx
  47:24  error  'data' is missing in props validation  react/prop-types
```